src/components/classes/Grid.ts

```diff
@@ -250,7 +261,7 @@ export default class Grid {
             let child_dna: Vector[] = [];
 
             for(let i = 0; i < min_dna_length; i++) {
-                if(Math.random() < 0.01) child_dna.push({x: Math.random() * (Constants.ACC_MAX - Constants.ACC_MIN) + Constants.ACC_MIN, y: Math.random() * (Constants.ACC_MAX - Constants.ACC_MIN) + Constants.ACC_MIN}); // TODO: implement mutation rate slider AND create a vector library
+                if(Math.random() < 0.001) child_dna.push({x: Math.random() * (Constants.ACC_MAX - Constants.ACC_MIN) + Constants.ACC_MIN, y: Math.random() * (Constants.ACC_MAX - Constants.ACC_MIN) + Constants.ACC_MIN}); // TODO: implement mutation rate slider AND create a vector library
```

Comment:
Do we want to keep this so low?

Reply:
yes seems to work pretty well, too high is too random